crates/common/trie/benches/formatter.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
should we move this to .github/scripts?

Reply:
It's a good Idea, i'll try it out and see how it goes.